src/featureFlags/evaluation.test.ts

```diff
@@ -126,26 +128,26 @@ describe("featureFlags/evaluation.ts", function () {
       reason: fakeReason,
     };
     FeatureFlags[FeatureFlag.GLOBAL_DISABLED_VERSIONS] = [disabledVersion];
+    clientVariationStub.withArgs(FeatureFlag.GLOBAL_DISABLED_VERSIONS).returns([disabledVersion]);
 
     const reason: string | undefined = checkForExtensionDisabledReason();
 
-    assert.ok(reason);
     assert.strictEqual(reason, disabledVersion.reason);
   });
 
   it("checkForExtensionDisabledReason() should return 'Unspecified reason' when disabled version has no reason", function () {
     // globally enabled, current version disabled but missing reason
     FeatureFlags[FeatureFlag.GLOBAL_ENABLED] = true;
     const disabledVersion: any = {
-      product: "vscode",
+      product: env.uriScheme,
```

Comment:
This was the culprit. Local testing wasn't specifically using the `VSCODE_VERSION` environment variable ([like our pipeline does](https://github.com/confluentinc/vscode/blob/7aeb2354158cfb2781263d46853fdc8f664f975c/.semaphore/semaphore.yml#L49-L68)) so our local Mocha test runner is using `stable` by default, which would incorrectly match on `vscode` when run through a VS Code Insiders terminal.